### DISCUSSION TITLE
🎉  New Source: Goldcast

### DISCUSSION THREAD

**Discussion by FVidalCarneiro** [airbytehq/airbyte#38785] [New Connector Request]
[Goldcast](https://www.goldcast.io/) is a SaaS for event organization, namely, webinars, virtual events and field events. Goldcast has an API allowing customers to ingest data related to them (documentation: https://customapi.goldcast.io/swagger-ui/#/).

This issue requests for a new airbyte connector to be developed. This can be done using the [low-code connector framework](https://docs.airbyte.com/connector-development/config-based/low-code-cdk-overview).

**Comment by FVidalCarneiro** [comment]
Pull request created to address issue [here](https://github.com/airbytehq/airbyte/pull/38786/files).

**Comment by natikgadzhi** [comment]
Done!
